packages/terra-compact-interactive-list/src/subcomponents/Row.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import ThemeContext from 'terra-theme-context';
+import classNames from 'classnames/bind';
+import styles from './Row.module.scss';
+import Cell from './Cell';
+import cellShape from '../proptypes/cellShape';
+import columnShape from '../proptypes/columnShape';
+import { widthUnitTypes } from '../utils/constants';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * An identifier to uniquely identify the row.
+   */
+  id: PropTypes.string.isRequired,
+
+  /**
+   * Data to be displayed in the cells of the row. Cells will be rendered in the row in the order given.
+   */
+  cells: PropTypes.arrayOf(cellShape),
+
+  /**
+   * Data for columns.
+   */
+  columns: PropTypes.arrayOf(columnShape).isRequired,
+
+  /**
+   * A number for column minimum width. Defaults to 60.
+   */
+  columnMinimumWidth: PropTypes.number,
+
+  /**
+   * A number for column minimum width.
+   */
+  columnMaximumWidth: PropTypes.number,
+
+  /**
+   * A number of visual columns.
+   */
+  numberOfColumns: PropTypes.number.isRequired,
+
+  /**
+   * Indicates if the column is flex growing column (can grow wider or shrink).
+   */
+  isResponsive: PropTypes.bool.isRequired,
+
+  /**
+   * Row's width in units set by widthUnit prop, such as `px`, `em`, or `rem`.
+   * Disregarder if flexGrow is set to true.
+   */
+  rowWidth: PropTypes.number,
+
+  /**
+   * Row's maximum width for flex growing rows in units set by widthUnit prop, such as `px`, `em`, or `rem`.
+   * Will have no effect if the width is set and flexGrow is omitted or false.
```

Comment:
Should this comment say 'if row width is set'?
Also there should be more information on where the flexGrow property reference is from.

Reply:
Fixed in https://github.com/cerner/terra-framework/pull/1869/commits/6d4771b2bc54be722bb4c4efeca3d218192a205b